i18n/en.json

```diff
@@ -262,6 +262,11 @@
   "admin.billing.subscription.cancelSubscriptionSection.contactUs": "Contact Us",
   "admin.billing.subscription.cancelSubscriptionSection.description": "At this time, deleting a workspace can only be done with the help of a customer support representative.",
   "admin.billing.subscription.cancelSubscriptionSection.title": "Cancel your subscription",
+  "admin.billing.subscription.cloudTrial.2daysLeftOnTrial": "There are 2 days left on your free trial",
+  "admin.billing.subscription.cloudTrial.3daysLeftOnTrial": "There are 3 days left on your free trial",
```

Comment:
I believe we can consolidate these twos and have it dynamically show the amount of days